packages/sources/sportsdataio/src/sport/nfl/index.ts

```diff
@@ -17,12 +17,18 @@ export const execute: ExecuteWithConfig<Config> = async (request, context, confi
   const endpoint = validator.validated.data.endpoint
 
   switch (endpoint.toLowerCase()) {
+    case currentSeason.NAME: {
+      return await currentSeason.execute(request, context, config)
+    }
     case schedule.NAME: {
       return await schedule.execute(request, context, config)
     }
     case scores.NAME: {
       return await scores.execute(request, context, config)
     }
+    case teams.NAME: {
+      return await teams.execute(request, context, config)
+    }
```

Comment:
Can we add documentation of these new endpoints in the README?

Reply:
Done.